examples/basic-tracer-node/index.js

```diff
@@ -22,13 +22,13 @@ if (EXPORTER.toLowerCase().startsWith('z')) {
   exporter = new CollectorExporter(options);
 }
 
-const registry = new BasicTracerRegistry();
+const provider = new BasicTracerProvider();
 
 // Configure span processor to send spans to the provided exporter
-registry.addSpanProcessor(new SimpleSpanProcessor(exporter));
+provider.addSpanProcessor(new SimpleSpanProcessor(exporter));
 
-// Initialize the OpenTelemetry APIs to use the BasicTracerRegistry bindings
-opentelemetry.initGlobalTracerRegistry(registry);
+// Initialize the OpenTelemetry APIs to use the BasicTracerProvider bindings
+opentelemetry.initGlobalTracerProvider(provider);
 const tracer = opentelemetry.getTracer('example-basic-tracer-node');
```

Comment:
This should be opentelemetry.**trace**.getTracer('example-basic-tracer-node');, you need to update this in almost all the examples.

Reply:
Fixed everywhere